contracts/testing/ExposedVotingProxy.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../VotingProxy.sol";
+
+contract ExposedVotingProxy is VotingProxy {
+    constructor(IERC20Metadata _DPS, BallotTagManager _ballotTagManager) VotingProxy(_DPS, _ballotTagManager) {}
+
+    function _delegates(address to, uint32 tagIndex) external view returns(address[] memory) {
+        address[] memory proxies = new address[](delegates[to][tagIndex].grantCount);
+        for(uint32 i = 0; i < delegates[to][tagIndex].grantCount; i++) {
+            proxies[i] = delegates[to][tagIndex].indexVoter[i];
+        }
+        return proxies;
+    }
+
+    function _proxyVoters(address from, uint32 tagIndex) external view returns(address) {
+        return proxyVoters[from][tagIndex];
+    }
+}
```

Comment:
Make those properties public, because users will need to know to whom they've delegated their vote and who delegated to them. Since they will be marked as view, there is now way to obfuscate the data to all users except the delegator/voter.